src/driver/device.cc

```diff
@@ -1,10 +1,96 @@
+#include <cstring>
 #include <driver/device.h>
-#ifdef FT_WITH_CUDA
 #include <driver/gpu.h>
-#endif
-
 namespace freetensor {
 
+const size_t deviceNameMaxLength = 255; // not 256, for '\0' terminated string
```

Comment:
`static const size_t deviceNameMaxLength = 255;`, since it is local to this file.